### PR TITLE
Adding pop up and html dynamically

### PR DESCRIPTION
**Pull request by Sammy3000**
## In this pull request I did the following:
- When the user clicks (or taps) the button to check project details, the popup with details about the project appears.
- When the user clicks (or taps) the close (X) button, the popup disappears.

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -798,6 +1006,8 @@ h3 {
   }
 }
 
+/* modal window desktop */
+
```
Comment (Abenezer-Tilahun) [inline style.css:1035]: - [ ]  As stated in the project requirement Kindly make sure that the popup is responsive, it fits mobile and desktop sizes properly.So Kindly fix the responsiveness for the mobile screen size.
- Your Project Pop-up for mobile screen size Look likes this : 

Capture

-  The Figma Pop-up for Mobile Screen size Look likes this : 

Capture

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @Sammy3000 , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git-Flow.
✔️ Professional README
✔️ Good commit messages

almost

## Required Changes ♻️

- [ ] Kindly I recommend you to update your linters.yml file with the [JavaScript](https://github.com/microverseinc/linters-config/tree/master/javascript) linters.yml file.

https://github.com/Sammy3000/Sammy3000.github.io/blob/f9e768408518facc3a3c2223a499c6b56e037af0/.github/workflows/linters.yml#L1-L56

_Check the comments below the review._
## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: style.css [hunk 2]
```diff
@@ -592,6 +739,85 @@ h3 {
...
+    justify-content: center;
+    background: rgba(193, 199, 208, 0.5);
+    z-index: 200;
+    backdrop-filter: blur(5px);
+  }
+
+  .modal-content {
+    margin: 5rem 3.5rem 0 3.5rem;
+  }
+
```
Comment (GedeonTS) [inline style.css]: - [ ] Good job adding the js linters checks as required by the previous reviewer, :clap: 
Kindly make sure you stick to the design by respecting the proportion of the size of the of the image and it's margins. on both desktop and mobile version.

- [ ] Again like the previous reviewer suggested in his illustration please make sure you fix the buttons designs to meet the Figma design, and also adding the close button on mobile as it is not showing up.

## Your mobile version now:

- mobile
yourMob
- desktop

youDesk

Figma designs:
- mobile
figMob
- desktop
figDesk

**Review by GedeonTS — CHANGES_REQUESTED** [review]
# STATUS: CHANGES REQUIRED :recycle: :
Hi @Sammy3000 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



 ## Highlights :tophat: :+1: 
- Linters are passing :+1: 
- The details popup is appearing :+1: 
- Good use of GitHub flow :+1: 
 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions :ringed_planet: 
- [OPTIONAL] Kindly make sure you write good descriptive commit messages, always starting with a capital letter ,in imperative form :thinking: 
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by franclobo — APPROVED** [review]
Hi @Sammy3000, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights

- No linter errors ✔️ 
- Correct Github flow ✔️ 
- Descriptive PR ✔️ 

## Optional suggestions
- **[OPTIONAL]** Kindly correct the distance between the name of the action button and its icon 👇🏽 

| Figma design | Your design |
|---|---|
| image | image |


_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**